src/components/TheTopNavbar.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <v-toolbar data-cy="nav-drawer" color="primary">
+    <v-toolbar-title>
+      <div class="d-flex flex-md-row flex-row-reverse align-center justify-space-between" style="cursor: pointer">
+        <v-menu  transition="slide-x-transition">
+          <template #activator="{ props }">
+            <v-btn
+              data-cy="user-menu"
+              class="hidden-xs-only d-flex text-body-1"
+              v-bind="props"
+              variant="text"
+            >
+              <v-icon icon="mdi-account-circle" />
+              {{$store.state.auth.username}}
+            </v-btn>
+          </template>
+          <v-list density="compact" class="bg-surface-2 text-surface-1">
+            <v-list-item
+              v-for="({ text, icon, page }, i) in menuItems"
+              :key="i"
+              :prepend-icon="`mdi-${icon}`"
+              :title="text"
+              :to="page"
+              exact
+              :data-nav="text"
+            />
+          </v-list>
+        </v-menu>
+        <img
+          id="logo"
+          alt="Cuttle logo"
+          src="/public/img/cuttle_logo_text_white.svg"
```

Comment:
I believe we should omit /public (see other logo instances e.g. in the login page)
```suggestion
          src="/img/cuttle_logo_text_white.svg"
```